ios/FluentUI/Controls/SegmentedControl.swift

```diff
@@ -366,38 +475,69 @@ open class SegmentedControl: UIControl {
         }
     }
 
-    private func createButton(withTitle title: String) -> SegmentedControlButton {
+    private func createTabButton(withTitle title: String) -> SegmentedControlButton {
         let button = SegmentedControlButton(style: style)
         button.setTitle(title, for: .normal)
         button.accessibilityLabel = title
         button.addTarget(self, action: #selector(handleButtonTap(_:)), for: .touchUpInside)
         return button
     }
 
-    @objc private func handleButtonTap(_ sender: SegmentedControlButton) {
+    private func createSwitchButton(withTitle title: String) -> PillButton {
+        let pillStyle = style == .onBrandPill ? PillButtonStyle.onBrand : PillButtonStyle.primary
+        let button = PillButton(pillBarItem: PillButtonBarItem(title: title), style: pillStyle)
+        button.setTitle(title, for: .normal)
+        button.accessibilityLabel = title
+        if #available(iOS 13, *) {
```

Comment:
shouldn't need this. the min is already ios13

Reply:
I was getting errors from Xcode when I tried to build without the available check. But it looks like it's working without it now, so I'll remove and update.